docs/pages/preview/upcoming-releases.mdx

```diff
@@ -10,21 +10,34 @@ The teleport team delivers a new major release roughly every 3 months.
 
 ### Teleport 9.1
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for the CLI and the Web UI.
 
 | Version     | Date             |
 |-------------|------------------|
-| Release     | Early April 2022 |
+| Release     | April 18th 2022  |
```

Comment:
Will we have something ready for April 18th? I thought we were targeting April 29th now?